src/editbookmarksdialog.cpp

```diff
@@ -76,6 +76,8 @@ void EditBookmarksDialog::accept() {
             }
             QString name = item->data(0, Qt::DisplayRole).toString();
             QUrl url = QUrl::fromUserInput(item->data(1, Qt::DisplayRole).toString());
+            if (!url.isValid())
+                url = QUrl("file:///");
```

Comment:
IMO, user's home is a better option:
```c++
            if (!url.isValid()) {
                url = QUrl::fromUserInput(QString::fromUtf8(Fm::FilePath::homeDir().toString().get()));
            }
```

Reply:
agreed

Reply:
Please also keep the code style. I didn't above (it should have been `if(...)`).

Reply:
I have updated the PR